2018-09-20 15:05:22.609 [main] ERROR com.csy.o2o.service.impl.LocalAuthServiceImpl - nested exception is org.apache.ibatis.binding.BindingException: Parameter 'updateTime' not found. Available parameters are [password, name, newPassword, param3, userID, param4, param1, param2]
2018-09-20 15:08:20.916 [main] ERROR com.csy.o2o.service.impl.LocalAuthServiceImpl - nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
### Cause: org.springframework.jdbc.CannotGetJdbcConnectionException: Could not get JDBC Connection; nested exception is java.sql.SQLException: Connections could not be acquired from the underlying database!
2018-09-20 15:09:17.343 [main] DEBUG com.csy.o2o.dao.LocalAuthDao.updatePasswd - ==>  Preparing: update tb_local_auth SET password =?, update_time=? where user_id =? and user_name=? and password=? 
2018-09-20 15:09:17.383 [main] DEBUG com.csy.o2o.dao.LocalAuthDao.updatePasswd - ==> Parameters: csy665789987(String), 2018-09-20 15:09:16.869(Timestamp), 1(Long), 老王(String), 123456(String)
2018-09-20 15:09:17.392 [main] DEBUG com.csy.o2o.dao.LocalAuthDao.updatePasswd - <==    Updates: 1
